packages/python/chart-studio/chart_studio/grid_objs/grid_objs.py

```diff
@@ -199,9 +197,7 @@ def __init__(self, columns_or_json, fid=None):
                         )
             # collect and sort all orders in case orders do not start
             # at zero or there are jump discontinuities between them
-            all_orders = []
-            for column_name in columns_or_json["cols"].keys():
-                all_orders.append(columns_or_json["cols"][column_name]["order"])
+            all_orders = [columns_or_json["cols"][column_name]["order"] for column_name in columns_or_json["cols"].keys()]
```

Comment:
I think this can be simplified to:
```python
all_orders = [D['order'] for D in columns_or_json["cols"].values()]
```
but, I'm not sure what `columns_or_json` looks like. I guess it's a dictionary.

Reply:
You are right. Please dont mix single and double quotes :)

Reply:
You are right, I changed this there and below.